### DISCUSSION TITLE
Hide input, output, turn off download in Argo UI

### DISCUSSION THREAD

**Discussion by thewalldao** [argoproj/argo-workflows#11136] [Q&A] [answered]
Hi guys,
I want to hide input, output, and turn off download shown on argo UI because that content contains credential.
How I can do that ?

**Comment by agilgur5** [comment, marked as answer]
Argo's authorization model is based entirely on Kubernetes RBAC, including the UI.
So to prevent users from seeing certain parts of the UI, you can limit their RBAC. For instance, not giving `pods/log` permissions so that they are unable to view logs.

See #6490 for an enhancement request for more granular RBAC via Casbin (similar to Argo CD).
There are also a couple issues about hiding or graying out UI buttons when permissions are not available, such as #6243
